include/emp/base/error.hpp

```diff
@@ -14,9 +14,44 @@
 
 #include <iostream>
 #include <string>
+#include <sstream>
 
 namespace emp {
 
+/// TDEBUG should trigger its EMP equivalent.
+#ifdef TDEBUG
+#define EMP_TDEBUG 1
+#endif
+
+// If we're testing, do NOT abort, and instead save all data so it can be tested
+#ifdef EMP_TDEBUG
```

Comment:
Could we split out the implementations to different header files like we have for i.e., _tdebug_error_trigger.hpp and _error_trigger.hpp